app/pages/adduser/template.njk

```diff
@@ -0,0 +1,53 @@
+{% extends 'includes/layout.njk' %}
+{% from 'components/back-link/macro.njk' import backLink %}
+{% from 'components/button/macro.njk' import button %}
+{% from 'components/label/macro.njk' import label %}
+{% from 'components/view-data-text/macro.njk' import viewDataText %}
+{% from 'components/text-field/macro.njk' import textField %}
+{% from '../../common/components/table.njk' import table %}
+
+{% block body %}
+  <div class="nhsuk-width-container" data-test-id="add-user-page">
+    <div data-test-id="go-back-link">
+      {{ backLink({
+        "href": "./",
+        "text": "Back"
+      }) }}
+    </div>
+    <h1 data-test-id="add-user-page-title" class="nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3">{{ title }}</h1>
+    <h2 data-test-id="add-user-page-description" class="nhsuk-u-font-size-22 nhsuk-u-margin-bottom-6">{{ description }}</h2>
+
+    <h3 data-test-id="org-name-subheading" class="nhsuk-u-font-weight-bold nhsuk-u-margin-bottom-2">{{ orgNameSubheading }}</h3>
+    {{ viewDataText({
+      dataTestId: "org-name",
+      classes: "nhsuk-u-margin-bottom-5",
+      data: orgName
+    }) }}
+
+    <form method="post" action="{{ submitActionUrl }}">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-two-thirds">
+          {% for question in questions %}
+            <div data-test-id="add-user-page-{{ question.id }}">
+              {{ label({
```

Comment:
can use the stuff inside textField to show this label.
params.question.mainAdvice = text
put classes on div

Reply:
So I put this in but the label text size is too large and bold so I can either add class to outer div and use it to force the styles to that component or use the label component as it is